src/modules/outliers/index.js

```diff
@@ -0,0 +1,89 @@
+import { getZScoreHelper, Z_SCORE } from './zScore'
+import { getModZScoreHelper, MODIFIED_Z_SCORE } from './modZScore'
+import { getIQRHelper, IQR } from './iqr'
+import { normalizerMap, XY_RATIO } from './normalization'
+import { getXYStats } from './xyStats'
+
+export const THRESHOLD_FACTOR = 1.5
+
+export const defaultConfig = {
+    thresholdFactor: THRESHOLD_FACTOR,
+    normalizationMethod: XY_RATIO,
+    outlierMethod: IQR,
+    percentile: 1,
+}
+
+const getDataWithNormalization = (data, normalizationMethod) => {
+    const normalizer = normalizerMap[normalizationMethod]
+
+    if (typeof normalizer !== 'function') {
+        throw `Normalization method ${normalizationMethod} not supported`
+    }
+
+    return data
+        .map(point => ({
+            point,
+            normalized: normalizer(point),
+        }))
+        .sort((a, b) => (a.normalized < b.normalized ? -1 : 1))
+}
+
+const getPercentiles = (percentile, xyStats) => {
+    const xPercentileValue = xyStats.xSum * (percentile / 100)
+    const yPercentileValue = xyStats.ySum * (percentile / 100)
+
+    return [
+        {
+            name: `${percentile}% of Total X Values`,
```

Comment:
Should this name use `i18n`?

Reply:
Also applies to: 
https://github.com/dhis2/analytics/pull/791/files#diff-0a57656b857d5f0d54d896779c950a3858a6eddda1ec37331f9842e707a1a210R89
https://github.com/dhis2/analytics/pull/791/files#diff-0a57656b857d5f0d54d896779c950a3858a6eddda1ec37331f9842e707a1a210R134
https://github.com/dhis2/analytics/pull/791/files#diff-47ad76a2032009b8ae25aa31a4978f146a9f243196cd71f9626f2f56e38d7a89R50